src/callbacks/callbacks.jl

```diff
@@ -246,7 +246,7 @@ NVTX.@annotate function rrtmgp_model_callback!(integrator)
 
     set_surface_albedo!(Y, p, t, p.atmos.surface_albedo)
 
-    RRTMGPI.update_fluxes!(rrtmgp_model)
+    RRTMGPI.update_fluxes!(rrtmgp_model, UInt32(t / integrator.p.dt))
```

Comment:
So for nightly tiny AMIP runs we want this to be deterministic but different among the 4 runs? Would it be possible to make that also a function of slurm `job_id` or something that could be automatically different between the 4 runs?

Reply:
I think for nightly amip we will just have the default `radiation_seed_rng: false`. We only set it to true when debugging a restarting run.

Reply:
Ok, makes sense!

Reply:
Oh, just saw @Sbozzolo's comments. Why would this change the behavior when using `radiation_seed_rng: false`?

Reply:
> Oh, just saw @Sbozzolo's comments. Why would this change the behavior when using `radiation_seed_rng: false`?

It wouldn't, but I think it would be preferable to set this to true for our current AMIPs. This would allow us to directly use one of the restarts for debugging. When this is false, restarting a simulation leads to a different evolution.

Reply:
Sounds good. The nightly amip now has one deterministic (unique_seed: false) and three random runs. I think most of the time we will use the deterministic run for debugging, so maybe we can set it to true for the deterministic run only. Adding a perturbation to the initial condition is a good idea though, as we will need it at some point.

Reply:
>  I think most of the time we will use the deterministic run for debugging, so maybe we can set it to true for the deterministic run only.

Sounds good!